internal/orchestrator/ray/dashboard/serve.go

```diff
@@ -87,21 +87,22 @@ func EndpointToApplication(endpoint *v1.Endpoint, modelRegistry *v1.ModelRegistr
 	maps.Copy(args, endpoint.Spec.Variables)
 
 	app := RayServeApplication{
-		Name:        endpoint.Metadata.Name,
-		RoutePrefix: fmt.Sprintf("/%s", endpoint.Metadata.Name),
+		Name:        fmt.Sprintf("/%s/%s", endpoint.Metadata.Workspace, endpoint.Metadata.Name),
```

Comment:
Does the name need to include the workspace? Also, can this name pass Ray's verification?

Reply:
still has '/' in name